### PR TITLE
Completed patient about page

### PR DESCRIPTION
**Pull request by Hiperultimate**

### CODE REVIEW

File: client/src/components/patientComponents/contactInfo/contactInfo.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+.contact-single-field {
+  margin: 1em 0em;
+}
+
+.contact-grid {
+    background-color: white;
+    border-radius: 10px;
+  display: grid;
+  grid-template-columns: 1fr 0.75fr;
```
Comment (DamFlancoz) [inline client/src/components/patientComponents/contactInfo/contactInfo.css:19]: indentation mismatch

File: client/src/App.js [hunk 2]
```diff
@@ -18,7 +19,8 @@ function App() {
           <Login />
         </Route>
         <Route path="/about">
-          <DoctorAbout />
+          {/* <DoctorAbout /> */}
+          <PatientAbout />
         </Route>
```
Comment (DamFlancoz) [inline client/src/App.js:24]: remember to fix it later on

File: client/src/pages/patientAbout/PatientAbout.jsx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+                medicalHistory={[
+                  {
+                    DiseaseName: "Blood Cancer",
+                    CureTimeTaken: "21 Sept, 2021 - 30 December 2022",
+                    Comments: "Severe cases of blood ...",
+                    Medication: [
+                      {
+                        medName: "Chemo",
+                        medFrequency: "Once a day",
+                        medDuration: "31 Oct - 30 Dec",
```
Comment (DamFlancoz) [inline client/src/pages/patientAbout/PatientAbout.jsx:53]: would be better if you declare dummy data in function and then pass it here.

**Review by DamFlancoz — CHANGES_REQUESTED** [review]

File: client/src/App.js (hunk 2 above)
Comment (Hiperultimate) [reply client/src/App.js:24]: I think this will be fixed when we will fetch from the database.

**Pull request merged**